data/kamailio.cfg

```diff
@@ -1268,9 +1552,13 @@ route[SEND_ES_STATS] {
 	$http_req(hdr) = "Content-Type: application/json; charset=UTF-8";
 
 	# Index+Type defined in bulk object
-	$var(elasticurl) = "ELASTICSEARCH_URL/_bulk";
+	$var(elasticurl) = "http://YOUR_GRAYLOG_IP:12201/gelf";
```

Comment:
should $var(elasticurl) also be conditional and retain support both GRAYLOG and existing ES?